### ISSUE TITLE
It isn't clear that writing elements like <apple> is invalid html

### ISSUE THREAD

**Issue by Dan503** [flukeout/css-diner#107]
Great little learning game!

I can see why you have written the html the way you have. It make it simpler and easier to grasp the concept of CSS selectors.

For the people just starting out with web development though, they'll probably see the html in the game and think that writing up html like <sidebar> and <banner> is perfectly fine then get confused when their html doesn't work properly because of invalid markup.

There needs to be a clear message somewhere saying that writing html like <apple> was for the purposes of making CSS selectors more understandable and is actually invalid HTML and should not be done in real projects.

**Comment by wtyerogers** [comment]
> and is actually invalid HTML and should not be done in real projects.

True for beginners, false for the creation of custom HTML elements ;)

**Comment by Dan503** [comment]
> True for beginners

This games primary target market are beginners.

**Comment by flukeout** [comment]
@Dan503 I hear you - the benefit of using the fake food tags is that they have a nice visual counterpart on the table. I thought about using `<div class=`apple`></div>` for example, but that would really clutter things up and make it harder to demonstrate the distinction between tag name and class selectors. 

Does that change your thinking at all?

**Comment by Dan503** [comment]
No, I _like_ that it uses `<apple></apple>`, it's very clear and easy to grasp.

It needs a disclaimer at the beginning though explaining that the mark up used in the tutorial is invalid for the sake of clarity. Give one example of what valid mark up and css would look like on the disclaimer screen saying that this is what valid mark up would look like and show the comparison to the mark up used in the game.

After the disclaimer screen, you can use the invalid mark up for the actual game part of it.

It's just to stop new developers from thinking that the invalid mark up is fine to use in real world projects without sacrificing the clarity that the invalid mark up provides to the game.

**Comment by flukeout** [comment]
Okay, yeah. I'm wary of a disclaimer screen or anything that would impede the flow of the experience, but there are a couple of places I could add this info..
1. The "Need Help" screen available on the first level.
2. Some disclaimer or popup link from within the "editor" where the code is present.
3. Something dismissable in the footer maybe?

Thoughts?
